courses/machine_learning/deepdive2/structured/labs/4b_keras_dnn_babyweight.ipynb

```diff
@@ -0,0 +1,574 @@
+{
```

Comment:
<p>The "In this lab,..." paragraph feels very run-on to me. I'd suggest breaking it up into multiple sentences. Something along the lines of</p><p> </p><p>"We'll start by defining the CSV column names, label column and column defaults for our data inputs. Then, we'll construct a tf.data Dataset of features and the label from the CSV files and create inputs layers for the raw features. Next, we'll set up feature columns for the model inputs and build a deep neural network in Keras. We'll create a custom evaluation metric and build our DNN model and finally train and evaluate our model."</p><p> </p><p>If re-written, adjust in solution notebook to match.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/672/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>Thanks, ya I was trying to match my other notebooks of having the learning objectives in that sentence. But this notebook has a ton of them so it did get rather long. Changed to close to what you suggested!</p>